solutions/12/script.js

```diff
@@ -0,0 +1,89 @@
+const music = [["Yummy", "Justin Bieber"], ["Life Is Good", "Future featuring Drake"], ["Godzilla", "Eminem featuring Juice Wrld"], ["Don't Start Now", "Dua Lipa"], ["Intentions", "Justin Bieber featuring Quavo"], ["Blinding Lights", "The Weeknd"], ["On", "BTS"], ["Stupid Love", "Lady Gaga"], ["Baby Pluto", "Lil Uzi Vert"], ["Lo Mein", "Lil Uzi Vert"], ["Silly Watch", "Lil Uzi Vert"], ["Adore You", "Harry Styles"], ["Say So", "Doja Cat"]];
```

Comment:
Anti pattern
You shouldn't add value to index
, create array with songs object, example:
`{
 title: "Kiedy byłem wielorybem",
author: "Mc Wokay"
}`

Reply:
👍